transformations/subject_object_switch/test.json

```diff
@@ -0,0 +1,185 @@
+{
+    "type": "subject_object_switch",
+    "test_cases": [
+        {
+            "class": "SentenceSubjectObjectSwitch",
+            "inputs": {
+                "sentence": "Andrew has not returned the French book to the library."
+            },
+            "outputs": [
+                {
+                    "sentence": "the French book has not returned Andrew to the library."
```

Comment:
One suggestion, The first letter of the sentence should be in the capital. (the --> The). At the time of swapping we can check for the title() case.

Reply:
I fixed the sentence in the test. It made me think and I also added a basic check in the transformation, if the original sentence starts with a capital letter but the transformed one doesn't the first letter of the transformation is uppercased. This still leaves the problem of some initially uppercased letters ending up in the final transformed sentence, but I'm trying to keep the transformation as simple as I can. Thanks for your review.